wordle-web/pages/game.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <v-card class="mx-auto my-8" elevation="16" max-width="344">
+    <v-card-item>
+      <v-card-title> Hurdle Over This Wordle!!! </v-card-title>

```

Comment:
Clever name!

Reply:
Haha thank you! I like messing around with content/wording in my code when I can